_includes/shared/footer/subcript.gs

```diff
@@ -0,0 +1,20 @@
+var wbook = SpreadsheetApp.openByUrl(
+  "https://docs.google.com/spreadsheets/d/1lEtNFeTlXL4u5ZBpAFAGQRmPBdR_t8y3T2U-IJkpS90/edit#gid=0"
+);
+var sheet = wbook.getSheetByName("Sheet1");
+
+// Post a request to the specific link
+function doPost(e) {
+  var action = e.parameter.action;
+  if (action == "addUser") return addUser(e);
```

Comment:
Let's log an error when the action is wrong
e.g. else{
Logger.log("Unknown action: " + action);
}